docs/cvl/genericrules.md

```diff
@@ -0,0 +1,18 @@
+Builtin Rules
+=============
+CVT has builtin rules targeted at finding known vulnerabilities. 
```

Comment:
CVT is not an external name, use Certora Prover instead

Reply:
changed